backend/src/providers/transactions.ts

```diff
@@ -126,6 +128,68 @@ export const getTransactionInfo = async (
   return transaction[0] || null;
 };
 
+export const getTransactionDetails = async (
+  transactionHash: string
+): Promise<TransactionDetails | null> => {
+  const transactionBaseInfo = await getTransactionInfo(transactionHash);
+
+  if (!transactionBaseInfo) {
+    return null;
+  }
+  const transactionInfo = await sendJsonRpc("EXPERIMENTAL_tx_status", [
+    transactionBaseInfo.hash,
+    transactionBaseInfo.signerId,
+  ]);
+
+  const receiptsOutcome = transactionInfo.receipts_outcome.map((receipt) => ({
```

Comment:
We actually need only `tokensBurnt` from outcome here, I don't think we need handle around `proof` or `id` or `includedInBlock`.